app/views/admin/accounts/index.html.haml

```diff
@@ -2,7 +2,7 @@
   = t('admin.accounts.title')
 
 - content_for :header_tags do
-  = javascript_pack_tag 'admin', async: true, crossorigin: 'anonymous'
+  = vite_javascript_tag 'admin.jsx', crossorigin: 'anonymous'
```

Comment:
From what I can tell, we can still retain the `async: true` here, and it'll be passed through on the `<script>` tag